src/schema/nmdc.yaml

```diff
@@ -1524,24 +1517,27 @@ classes:
       - "sample mgnify link https://www.ebi.ac.uk/metagenomics/studies/MGYS00005757"
       - "GOLD, insdc.srs and mgnify are reasonable prefixes for alternative study identifiers, but no longer for the Study.id"
 
-  BiosampleProcessing:
-    aliases:
-      - material processing
+  MaterialProcessing:
     is_a: PlannedProcess
     description:
-      A process that takes one or more biosamples as inputs and generates
-      one or more biosamples as outputs. An example of an output includes samples cultivated from another
-      sample.
+      A process that takes one or more samples as inputs and generates
+      one or more samples as outputs. 
     slots:
       - has_input
+    notes: 
+      - This class is a replacement for BiosampleProcessing.
     slot_usage:
       id:
         required: true
         structured_pattern:
-          syntax: "{id_nmdc_prefix}:bsmprc-{id_shoulder}-{id_blade}{id_version}{id_locus}"
+          syntax: "{id_nmdc_prefix}:matprc-{id_shoulder}-{id_blade}{id_version}{id_locus}"
```

Comment:
Do we want to make this class abstract? Looks similar to DataGeneration where we don't intend to use the parent class.